module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -284,6 +284,19 @@ protected function initSearch($request)
         }
     }
 
+    /**
+     * Check whether a specific facet supports filtering
+     *
+     * @param string $facet The facet to check
+     *
+     * @return bool
+     */
+    public function supportsFacetFiltering($facet)
```

Comment:
It might be worth moving this method into VuFind\Search\Base\Params, and add the check for the `setFacetContains()` method into this method. That way, it won't be necessary to introduce a custom supportsFacetFiltering method into other backends if we want them to be supported in future, as long as they meet the prerequisites.

Reply:
Also note that if we move this to base, then you can just call it directly in the template instead of checking if it exists, since it will be guaranteed available everywhere.